po/fr.po

```diff
@@ -1,5 +1,723 @@
+msgid "Ellipse:"
+msgstr "Ellipse :"
+
+#: src/Dialogs/ShortcutsDialog.vala:92
+msgid "Text:"
+msgstr "Texte :"
+
+#: src/Dialogs/ShortcutsDialog.vala:94
+msgid "Image:"
+msgstr "Image :"
+
+#: src/Dialogs/ShortcutsDialog.vala:97
+#: src/Layouts/Partials/TransformPanel.vala:174
+msgid "Transform"
+msgstr "Transformer"
+
+#: src/Dialogs/ShortcutsDialog.vala:98
+msgid "Raise selection:"
+msgstr "Augmenter la sélection :"
+
+#: src/Dialogs/ShortcutsDialog.vala:100
+msgid "Lower selection:"
+msgstr "Diminuer la sélection :"
+
+#: src/Dialogs/ShortcutsDialog.vala:102
+msgid "Raise selection to top:"
+msgstr "Monter la sélection :"
+
+#: src/Dialogs/ShortcutsDialog.vala:104
+msgid "Lower selection to bottom:"
+msgstr "Descendre la sélection :"
+
+#: src/Dialogs/ShortcutsDialog.vala:106
+msgid "Flip horizontally:"
+msgstr "Retourner horizontalement :"
+
+#: src/Dialogs/ShortcutsDialog.vala:108
+msgid "Flip vertically:"
+msgstr "Retourner verticalement :"
+
+#: src/FileFormat/FileManager.vala:46
+msgid "Save Akira file"
+msgstr "Enregistrer un fichier Akira"
+
+#: src/FileFormat/FileManager.vala:48 src/Layouts/HeaderBar.vala:190
+msgid "Save"
+msgstr "Enregistrer"
+
+#: src/FileFormat/FileManager.vala:71
+msgid "Akira files"
+msgstr "Fichiers Akira"
+
+#: src/FileFormat/FileManager.vala:76
+msgid "All files"
+msgstr "Tous les fichiers"
+
+#: src/FileFormat/FileManager.vala:103
+msgid "Open Akira file"
+msgstr "Ouvrir un fichier Akira"
+
+#: src/FileFormat/FileManager.vala:105 src/Layouts/HeaderBar.vala:170
+msgid "Open"
+msgstr "Ouvrir"
+
+#: src/Layouts/HeaderBar.vala:69 src/Lib/Managers/ExportManager.vala:296
+msgid "Untitled"
+msgstr "Sans titre"
+
+#: src/Layouts/HeaderBar.vala:71
+msgid "Menu"
+msgstr "Menu"
+
+#: src/Layouts/HeaderBar.vala:75
+msgid "Insert"
+msgstr "Insérer"
+
+#: src/Layouts/HeaderBar.vala:82
+msgid "Group"
+msgstr "Grouper"
+
+#: src/Layouts/HeaderBar.vala:84
+msgid "Ungroup"
+msgstr "Dégrouper"
+
+#: src/Layouts/HeaderBar.vala:87
+msgid "Up"
+msgstr "Monter"
+
+#: src/Layouts/HeaderBar.vala:92
+msgid "Down"
+msgstr "Descendre"
+
+#: src/Layouts/HeaderBar.vala:97
+msgid "Top"
+msgstr "Devant"
+
+#: src/Layouts/HeaderBar.vala:102
+msgid "Bottom"
+msgstr "Derrière"
+
+#: src/Layouts/HeaderBar.vala:107
+msgid "Settings"
+msgstr "Paramètres"
+
+#: src/Layouts/HeaderBar.vala:118
+msgid "Difference"
+msgstr "Différence"
+
+#: src/Layouts/HeaderBar.vala:120
+msgid "Exclusion"
+msgstr "Exclusion"
+
+#: src/Layouts/HeaderBar.vala:122
+msgid "Intersect"
+msgstr "Intersection"
+
+#: src/Layouts/HeaderBar.vala:124
+msgid "Union"
+msgstr "Union"
+
+#: src/Layouts/HeaderBar.vala:161
+msgid "New Window"
+msgstr "Nouvelle fenêtre"
+
+#: src/Layouts/HeaderBar.vala:182
+msgid "Open Recent"
+msgstr "Ouvert récemment"
+
+#: src/Layouts/HeaderBar.vala:195
+msgid "Save As"
+msgstr "Enregistrer sous..."
+
+#: src/Layouts/HeaderBar.vala:204
+msgid "Quit"
+msgstr "Quitter"
+
+#: src/Layouts/HeaderBar.vala:237
+msgid "Artboard"
+msgstr "Plan de travail"
```

Comment:
I wonder if `espace de travail` would be a better translation of `artboard`, as `plan de travail` refers mainly to a physical object, but I might be wrong.